openstack/tools/func_test_tools/manual_functests_runner.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -eu
```

Comment:
can you add this file to tox.ini so that it gets tested by bashate

Reply:
a docstring might be good to explain why and when this is useful to use

Reply:
thanks, will do